mcp/src/main/java/io/modelcontextprotocol/client/McpAsyncClient.java

```diff
@@ -335,6 +346,26 @@ public Mono<Void> closeGracefully() {
 		return this.mcpSession.closeGracefully();
 	}
 
+	// --------------------------
+	// Utility Methods
+	// --------------------------
```

Comment:
It's a not some "utility" but a fundamental prerequisite to almost any operation, perhaps we can group it under "initialization methods"?